auslib/admin/views/scheduled_changes.py

```diff
@@ -91,10 +91,14 @@ def _post(self, sc_id, form, transaction, changed_by):
             # from the rule (aka, set as NULL in the db). The underlying Form
             # will have already converted it to None, so we can treat it the
             # same as a modification here.
-            if (request.json and k in request.json):
+            if request.json and k in request.json:
                 what[k] = v.data
 
         where = {"sc_id": sc_id}
+        completed_scheduled_change = self.sc_table.select(where=where, transaction=transaction, columns=["complete"])
+        if completed_scheduled_change and completed_scheduled_change[0].get("complete"):
+            return Response(status=403, response="Scheduled change already completed. Cannot modify now.")
```

Comment:
Similar to #229, this is something that should be enforced at the database level, to make sure that other things that call into it (that may exist in the future) don't have to reimplement this.